gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -3590,7 +3592,7 @@ subroutine GFS_physics_driver                                 &
                              Model%clam_deep,   Model%c0s_deep,                    &
                              Model%c1_deep,  Model%betal_deep, Model%betas_deep,   &
                              Model%evfact_deep, Model%evfactl_deep,                &
-                             Model%pgcon_deep,  Model%asolfac_deep)
+                             Model%pgcon_deep,  Model%asolfac_deep, wet_deep) !lzhang
 !           if (lprnt) print *,' rain1=',rain1(ipr)
```

Comment:
there are a lot of "!lzhang" in the new changes, I think if there is not necessary, then you can delete them or add more detailed description.

Reply:
I suggest to remove "lzhang".

Reply:
did it